config/auth.js

```diff
@@ -0,0 +1,34 @@
+/*eslint no-undef: "error"*/
+/*eslint no-console: ["error", { allow: ["warn", "error"] }] */
```

Comment:
do you actually use any `console.error` in your code and do you really need that exception?

Reply:
this was removed as well. I used that in some part of early version and forgot to remove it.